src/test_utils.ts

```diff
@@ -0,0 +1,12 @@
+import { assertEquals } from "./dev_deps.ts";
+
+// deno-lint-ignore ban-types
+type IsAny<T> = unknown extends T ? T extends {} ? T : never : never;
```

Comment:
We could likely leverage this type again for another test util (`assertEqualsAnyTape`) to verify that we're getting `any` when we actually expect it to meet that `UnknownRuntimeType`.